Comments.Test/Comments.Test.csproj

```diff
@@ -64,4 +65,14 @@
     </ItemGroup>
     <Copy SourceFiles="%(DiffAssertionsFilePath.FullPath)" DestinationFolder="$(OutputPath)" Condition="Exists('%(DiffAssertionsFilePath.FullPath)')" />
   </Target>
+
+  <Target Name="AddAssemblyAliasToReactiveAsync"
+          AfterTargets="ResolveAssemblyReferences"
+          Condition="'$(TargetFramework)' == 'netcoreapp2.1'">
+	  <ItemGroup>
+		  <ReferencePath Condition=" '%(FileName)' == 'System.Interactive.Async' ">
+			  <Aliases>reactive</Aliases>
+		  </ReferencePath>
+	  </ItemGroup>
+  </Target>
 </Project>
```

Comment:
Whats this?

Reply:
This is required to workaround overlap between System.Collections.Generic.IAsyncEnumerable in System.Runtime and System.Interactive.Async. That's why I was specifying System.Collections.Generic.IAsyncEnumerable earlier, to try and target the right library. After adding this, .net's then able to target the correct IAsyncEnumerable, which made the fully qualified path to it unnecessary.

This was came about when adding the feature switching library.